File: SocialPulse/SocialPulse.Core/Entities/Answer.cs
namespace SocialPulse.Core
{
    public class Answer : BaseEntity
    {
        public string Text { get; set; } = null!;
        public int AdminId { get; set; }
        public User Admin { get; set; } = null!;
        public int QuestionId { get; set; }
        public Question Question { get; set; } = null!;
    }
}

File: SocialPulse/UI/mobile_socialpulse/lib/models/answer.dart
import 'package:json_annotation/json_annotation.dart';

part 'answer.g.dart';

@JsonSerializable()
class Answer{
  int? id;
  String? text;
  int? adminId;
  int? questionId;

  Answer (this.id,this.text,this.questionId,this.adminId);

  factory Answer.fromJson(Map<String, dynamic> json) =>
      _$AnswerFromJson(json);

  Map<String, dynamic> toJson() => _$AnswerToJson(this);
}